tests/test_pywebpack.py

```diff
@@ -87,6 +88,14 @@ def test_project_no_scripts(brokenprj):
         project.buildall()
 
 
+def test_project_failed_build(simpleprj):
+    # Remove a file necessary for the build
+    os.unlink(os.path.join(os.path.dirname(simpleprj), 'index.js'))
```

Comment:
Since `simpleprj` does not define a scope, it is `function`. However, if that were to change in the future this could break other tests since it is not linked again.

I am not against leaving it as is, since it shouldn't change. However, it is breaking a fixture purposely 😅 Let's see what others think @ntarocco @fenekku @slint

Reply:
I see your point, it might happen true. In my opinion we can leave it like this and eventually fix it in the future.